src/Conversion/ONNXToKrnl/Math/RandomNormal.cpp

```diff
@@ -0,0 +1,74 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//===----------------- Matmul.cpp - Lowering Matmul Op --------------------===//
+//
+// Copyright 2019 The IBM Research Authors.
+//
+// =============================================================================
+//
+// This file lowers the ONNX Random Normal Operator to Krnl dialect.
+//
+//===----------------------------------------------------------------------===//
+
+#include "src/Conversion/ONNXToKrnl/ONNXToKrnlCommon.hpp"
+#include "src/Dialect/Krnl/KrnlHelper.hpp"
+#include "src/Dialect/ONNX/IndexExpr.hpp"
+#include "src/Dialect/ONNX/MLIRDialectBuilder.hpp"
+#include "src/Dialect/ONNX/ONNXShapeHelper.hpp"
+
+#include <time.h>
+
+using namespace mlir;
+
+#define DEBUG_TRACE 0
```

Comment:
I do not see this macro used.  Consider using LLVM_DEBUG infrastructure for traces.